src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -39,24 +41,25 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 	@HostListener('document:keydown', ['$event'])
 	async onkeydown(e: KeyboardEvent) {
 		if (this.cellModel?.currentMode === CellEditModes.SPLIT || this.cellModel?.currentMode === CellEditModes.MARKDOWN) {
+			const keyEvent = new StandardKeyboardEvent(e);
 			let markdownTextTransformer = new MarkdownTextTransformer(this._notebookService, this.cellModel);
-			if ((e.ctrlKey || e.metaKey) && e.key === 'b') {
+			if ((keyEvent.ctrlKey || keyEvent.metaKey) && keyEvent.keyCode === KeyCode.KEY_B) {
 				// Bold Text
 				DOM.EventHelper.stop(e, true);
 				await markdownTextTransformer.transformText(MarkdownButtonType.BOLD);
-			} else if ((e.ctrlKey || e.metaKey) && e.key === 'i') {
+			} else if ((keyEvent.ctrlKey || keyEvent.metaKey) && keyEvent.keyCode === KeyCode.KEY_I) {
 				// Italicize text
 				DOM.EventHelper.stop(e, true);
 				await markdownTextTransformer.transformText(MarkdownButtonType.ITALIC);
-			} else if ((e.ctrlKey || e.metaKey) && e.key === 'u') {
+			} else if ((keyEvent.ctrlKey || keyEvent.metaKey) && keyEvent.keyCode === KeyCode.KEY_U) {
 				// Underline text
 				DOM.EventHelper.stop(e, true);
 				await markdownTextTransformer.transformText(MarkdownButtonType.UNDERLINE);
-			} else if ((e.ctrlKey || e.metaKey) && e.shiftKey && e.key === 'k') {
+			} else if ((keyEvent.ctrlKey || keyEvent.metaKey) && e.shiftKey && keyEvent.keyCode === KeyCode.KEY_K) {
```

Comment:
Does `e.shiftKey` also need to be changed to `keyEvent.shiftKey`?

Reply:
Ah dangit...